Sources/Shared/Models/VIMLiveQuota.swift

```diff
@@ -26,6 +26,13 @@
 
 import Foundation
 
+public enum LiveQuotaStatus: String {
```

Comment:
Can we have this conform to `CaseIterable`? I know we don't need it for anything at the moment, but it could be useful down the road.